components/list/list-item-button-mixin.js

```diff
@@ -116,7 +116,9 @@ export const ListItemButtonMixin = superclass => class extends ListItemMixin(sup
 
 	_onButtonFocus(e) {
 		if (this._getDescendantClicked(e)) {
-			e.stopPropagation();
+			requestAnimationFrame(() => {
+				this._focusingPrimaryAction = false;
```

Comment:
We don't want the focus primary action focus ring when focus is on another focusable item in the list item content.